### DISCUSSION TITLE
Avoid Email burst and send notification digest instead

### DISCUSSION THREAD

**Discussion by karussell** [community/community#4286] [Projects and Issues]
This is basically the request #4075 but not weekly and for issues instead of discussions.

The idea is to solve the annoying Email burst that you get when an issue is very active (ping pong of comments within a few minutes). Instead of getting one Email notification per comment in an issue (or discussion) it would be better when github collects them and sends this out e.g. every 15 minutes.

It would be important to think about what happens if you already read the comments in that issue via mobile or web. It should be probably configurable if you get them as an Email again.

**Comment by lukaseder** [comment]
This would be particularly useful for:

- Specific accounts (e.g. I'd like a digest for dependabot emails)
- Specific repositories (e.g. I'd like a rare digest for dependabot emails from other repositories)

**Comment by jeffijoe** [comment]
It would be especially useful for PR comment responses.
